scripts/metricsgen/metricsdiff/metricsdiff_test.go

```diff
@@ -0,0 +1,64 @@
+package main_test
+
+import (
+	"bytes"
+	"io"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	metricsdiff "github.com/tendermint/tendermint/scripts/metricsgen/metricsdiff"
+)
+
+func TestDiff(t *testing.T) {
+	for _, tc := range []struct {
+		name      string
+		aContents string
+		bContents string
+
+		expectedDiff metricsdiff.MetricsDiff
```

Comment:
This could just be `want` or `diff`, I don't think the expectation is a surprise 🙂